main/gen_gdl90.go

```diff
@@ -942,20 +925,6 @@ func updateStatus() {
 		globalStatus.GPS_solution = "Unknown"
 	}
 
-	if !(globalStatus.GPS_connected) || !(isGPSConnected()) { // isGPSConnected looks for valid NMEA messages. GPS_connected is set by gpsSerialReader and will immediately fail on disconnected USB devices, or in a few seconds after "blocked" comms on ttyAMA0.
```

Comment:
THis is now handled by the new GPSDeviceManager when it detects that there is no GPS connection, it will reset this info